server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/view/cassandra/configuration/ConfigurationEditedDTOTest.java

```diff
@@ -51,21 +52,21 @@ void fromShouldThrowWhenConfigurationAddedIsNull() {
 
     @Test
     void fromShouldReturnCorrespondingDTO() {
-        ConfigurationAdded configurationAdded = new ConfigurationAdded(
+        ConfigurationEdited configurationEdited = new ConfigurationEdited(
             EVENT_ID,
             CassandraMailQueueViewConfiguration.builder()
                 .bucketCount(BUCKET_COUNT)
                 .updateBrowseStartPace(UPDATE_PACE)
                 .sliceWindow(ONE_HOUR)
                 .build());
 
-        ConfigurationAddedDTO dto = ConfigurationAddedDTO.from(configurationAdded);
-        SoftAssertions softly = new SoftAssertions();
-        softly.assertThat(dto.getEventId()).isEqualTo(EVENT_ID_SERIALIZED);
-        softly.assertThat(dto.getType()).isEqualTo(CassandraMailQueueViewConfigurationModule.TYPE);
-        softly.assertThat(dto.getBucketCount()).isEqualTo(BUCKET_COUNT);
-        softly.assertThat(dto.getUpdateBrowseStartPace()).isEqualTo(UPDATE_PACE);
-        softly.assertThat(dto.getSliceWindow()).isEqualTo(ONE_HOUR);
-        softly.assertAll();
+
+        ConfigurationAddedDTO dto = ConfigurationAddedDTO.from(configurationEdited);
+        assertAll(
```

Comment:
please don't start using junit assertions.
You can use assertSoftly with a list of functions that are actual assertions.

Reply:
I'm the one actually guilty for that.

Do you have rationnals for this choice?

Reply:
Of course.
Assertj is a better assertion library (it's even written in junit 5 documentation).
We made the right choice in the past by adopting it. We should stick to it as much as possible.
That means avoiding to mix assertions from several libraries.
In this case, using `assertAll` is exactly the same as using assertSoftly like that : 
```
SoftAssertions.assertSoftly(softly -> {
            softly.assertThat(messageQuota.getUsed()).isEqualTo(QuotaCount.count(0));
            softly.assertThat(storageQuota.getUsed()).isEqualTo(QuotaSize.size(0));
        });
```
Often, having to use a given library forces us to learn it really.